be/src/env/env_hdfs.cpp

```diff
@@ -133,6 +133,20 @@ Status S3RandomAccessFile::readv_at(uint64_t offset, const Slice* res, size_t re
     return Status::InternalError("S3RandomAccessFile::readv_at not implement");
 }
 
+StatusOr<std::unique_ptr<NumericStatistics>> HdfsRandomAccessFile::get_numeric_statistics() {
+    struct hdfsReadStatistics* hdfs_statistics = nullptr;
```

Comment:
add `#ifndef BE_TEST` here, now BE_UT was not link with libhdfs.

Reply:
> add `#ifndef BE_TEST` here, now BE_UT was not link with libhdfs.

It's unneeded